<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>workspace</class>
 <widget class="Workspace" name="workspace">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>361</width>
    <height>283</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="fieldGrowthPolicy">
    <enum>QFormLayout::ExpandingFieldsGrow</enum>
   </property>
   <item row="1" column="0">
    <widget class="QLabel" name="selectedWorkflowNameLabel">
     <property name="text">
      <string>Selected</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="selectedWorkflowView">
     <property name="text">
      <string/>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="selectedWorkflowDescriptionLabel">
     <property name="text">
      <string>Description</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QTextBrowser" name="descriptionView">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>100</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="settingsLabel">
     <property name="text">
      <string>Settings</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QWidget" name="settingsWidget" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Workspace</class>
   <extends>QWidget</extends>
   <header>workspace.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
